runtime/darwinia/src/pallets/polkadot_xcm.rs

```diff
@@ -215,3 +221,63 @@ impl cumulus_pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type XcmExecutor = xcm_executor::XcmExecutor<XcmExecutorConfig>;
 }
+
+pub struct EthereumXcmEnsureProxy;
+impl xcm_primitives::EnsureProxy<AccountId> for EthereumXcmEnsureProxy {
+	fn ensure_ok(delegator: AccountId, delegatee: AccountId) -> Result<(), &'static str> {
+		// The EVM implicitely contains an Any proxy, so we only allow for "Any" proxies
```

Comment:
Could you explain this?

Reply:
This is copied from moonbeam, I don't really understand it, but it is used in [`transact_through_proxy`](https://github.com/PureStake/moonbeam/blob/master/pallets/ethereum-xcm/src/lib.rs#L193), we don't need it, I will disable this.

Reply:
@boundless-forest help review the `EthereumXcmEnsureProxy`.